package.json

```diff
@@ -75,6 +75,7 @@
     "browserify": "^17.0.0",
     "depcheck": "^1.4.7",
     "eslint": "^9.12.0",
+    "eslint-plugin-chai-friendly": "^1.0.1",
```

Comment:
I dont remember why this plugin is for, could we add it in the description why it is required?. eslint uses it directly? without referencing in the config?